src/main/java/log_in/interface_adapters/LogInControllerData.java

```diff
@@ -0,0 +1,20 @@
+package log_in.interface_adapters;
+
+// inputted data to be processed
+public class LogInControllerData {
```

Comment:
I think a few of us have data structures like this in our interface adapters so let's try to decide on a naming convention when we meet next time!